test/common/router/router_test.cc

```diff
@@ -723,7 +720,8 @@ TEST_F(RouterTest, AddCookieNoDuplicate) {
 
   EXPECT_CALL(callbacks_, encodeHeaders_(_, _))
       .WillOnce(Invoke([&](const Http::HeaderMap& headers, const bool) -> void {
-        EXPECT_EQ(headers.get(Http::Headers::get().SetCookie)->value().getStringView(), "foo=baz");
+        EXPECT_EQ(std::string{headers.get(Http::Headers::get().SetCookie)->value().getStringView()},
```

Comment:
`headers.getSetCookie()`?

Reply:
Good eye!
Sadly we can't change this one - setCookie is exempted from the O(1) headers list because it's not spec compliant to coalesce.

Reply:
oh! thanks